src/index.html

```diff
@@ -6,8 +6,47 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700;800&family=Roboto:wght@400;500;700&display=swap"
+     rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image"></div>
+
+      <div class="card__block">
+        <div class="text-Main">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+        <div class="text-Secondary">Product code: 195434</div>
```

Comment:
Need to use a semantic tags everywhere for text
```suggestion
        <h2 class="text-Main">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
        
        <p class="text-Secondary">Product code: 195434</p>
```